### DISCUSSION TITLE
API Auth procedure

### DISCUSSION THREAD

**Discussion by AdityaPrasad275** [civitai/civitai#679] [Q&A]
I am trying to get the models that i have favourited from my account and using the following api request code 
```
base_url = "https://civitai.com/api/v1/models"
params = {
    "limit" : 10,
    "favorites": True
}

headers = {
    "Authorization": f"Bearer {api_key}"
}
```

I am passing the correct api keyl but the result.json contains models that are not a part of my favourites. Infact the output is exactly the same as if i never used  `"favorites": True` 
I tried to play around with api on postman and in the authorization tab i selected api key as type, the name of the key as the key and the api key as value and the output is 
```
{
    "error": "Unauthorized"
}
```

What am i doing wrong 😭. Please guide me as to how to extract the favourited models/loras/everything from my account.

**Comment by AdityaPrasad275** [comment]
Just seconds after i posted this, i changed to "bearer token" type in postman and it worked! But the python code still doesn't work. It's as if favourites=true doesnt have any effect.

**Comment by AdityaPrasad275** [comment]
It was dumb mistake. I had to do `"favourites":"true"` instead of `True`  😑

**Reply by BlafKing** [reply]
Had the exact same issue, thx for posting the fix :)
